olp-cpp-sdk-dataservice-read/src/generated/api/BlobApi.cpp

```diff
@@ -31,46 +31,37 @@ namespace dataservice {
 namespace read {
 using namespace olp::client;
 
-CancellationToken BlobApi::GetBlob(
-    const OlpClient& client, const std::string& layerId,
-    const std::string& dataHandle, boost::optional<std::string> billingTag,
-    boost::optional<std::string> range,
-    const DataResponseCallback& dataResponseCallback) {
-  std::multimap<std::string, std::string> headerParams;
-  headerParams.insert(std::make_pair("Accept", "application/json"));
+BlobApi::DataResponse BlobApi::GetBlob(const client::OlpClient& client,
+                                       const std::string& layer_id,
+                                       const std::string& data_handle,
+                                       boost::optional<std::string> billing_tag,
+                                       boost::optional<std::string> range,
+                                       client::CancellationContext context) {
+  std::multimap<std::string, std::string> header_params;
+  header_params.insert(std::make_pair("Accept", "application/json"));
   if (range) {
-    headerParams.insert(std::make_pair("Range", *range));
+    header_params.insert(std::make_pair("Range", *range));
   }
 
-  std::multimap<std::string, std::string> queryParams;
-  if (billingTag) {
-    queryParams.insert(std::make_pair("billingTag", *billingTag));
+  std::multimap<std::string, std::string> query_params;
+  if (billing_tag) {
+    query_params.insert(std::make_pair("billingTag", *billing_tag));
   }
 
-  std::multimap<std::string, std::string> formParams;
+  std::multimap<std::string, std::string> form_params;
```

Comment:
Do we need to create an unused var? Instead, I would just pass a default constructed rvalue:

```cpp
client.CallApi(metadata_uri, "GET", query_params, header_params,
                     {}, nullptr, "", context);
```